### DISCUSSION TITLE
pc 深色能把标题栏改成深色吗

### DISCUSSION THREAD

**Discussion by amalaccs** [xfangfang/wiliwili#88] [Q&A]
白色看着好变扭😔

**Comment by xfangfang** [comment]
@amalaccs 
macOS 下是会根据系统主题自动切换的，Linux 我没注意过这个问题 （可能有的系统甚至都没有深浅主题这一回事）。
如果指的是 Windows 客户端的话，可以先参考这个 PR 临时解决一下：https://github.com/glfw/glfw/pull/2228

欢迎把改好的代码提交到 wiliwili 使用的 GLFW 仓库：https://github.com/xfangfang/glfw
我有时间会合并进来。

当然，我也可以改，不过因为我日常用 Windows 比较少，要等到我哪天时间比较富裕的，所以如果你急需这个功能，欢迎为 wiliwili 做贡献～

**Comment by xfangfang** [comment]
@amalaccs 可以下载测试版体验：https://github.com/xfangfang/wiliwili/actions/runs/8066530861
